packages/nextjs/components/scaffold-eth/Input/AddressInput.tsx

```diff
@@ -38,16 +41,20 @@ export const AddressInput = ({ value, name, placeholder, onChange, disabled }: C
     isSuccess: isEnsNameSuccess,
   } = useEnsName({
     address: settledValue as Address,
-    enabled: isAddress(debouncedValue),
     chainId: 1,
-    cacheTime: 30_000,
+    query: {
+      enabled: isAddress(debouncedValue),
+      gcTime: 30_000,
+    },
   });
 
   const { data: ensAvatar, isLoading: isEnsAvtarLoading } = useEnsAvatar({
-    name: ensName,
-    enabled: Boolean(ensName),
+    name: ensName ? normalize(ensName) : undefined,
```

Comment:
https://wagmi.sh/react/guides/migrate-from-v1-to-v2#removed-internal-ens-name-normalization

normalize is not called internally anymore.